instrumentation/net_ldap/lib/opentelemetry/instrumentation/net/ldap/instrumentation_service.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry/semconv/incubating/peer'
```

Comment:
I do not like this line; but I'll add a test to make sure we catch it when the constant is removed from `Incubating`

Reply:
https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1587/commits/111f18c81dd12f26cc0faa0d4599205148686638